bindings/python/examples/first_frame/first_frame.py

```diff
@@ -73,3 +74,10 @@
 
 image = np.array(frame.getData("depth"), copy=False)
 print(image)
```

Comment:
I guess it doesn't make sense to print to console the image since now is being displayed.